### ISSUE TITLE
Trigger for the Foscam camera

### ISSUE THREAD

**Issue by LesPaul1962** [edwinvdpol/com.foscam#19]
Why not create a motion trigger?

**Comment by edwinvdpol** [comment]
Hi,

The problem is, that the Foscam cameras are only support HTTP requests for getting data from the camera. If you want to enable something like a motion trigger, it means that Homey has to poll the camera every 3-5 seconds to see if the alarm is on. If you have for example 4 cameras available in Homey, the number of HTTP requests will be a lot.

I did some testing with 1 camera in the past, but it turned out that the memory usage of the app was rapidly growing over time. Not sure why though. I do not know if it is NodeJS, Homey or something else.

I want to provide a stable app which everyone can use. :wink:

That's why I decided not to implement this trigger.
